src/modules/core/components/Numeral/Numeral.css

```diff
@@ -50,10 +50,17 @@
   }
 }
 
+/* I see that in designs Dark is actually --dark not --grey-3 so maybe we
+should think about replacing this color here instead
+of adding one more theme option */
```

Comment:
Is `dark` theme used for Numeral a lot? I think it's a good idea not  to add too many themes. Maybe ask product about that? Usually, it's just an oversight on their part and they are happy to have less shades.